### ISSUE TITLE
Display the profile used when debugging in the console

### ISSUE THREAD

**Issue by nynexman4464** [johnste/finicky#177]
I'm trying to configure finicky to open certain URLs in a different google chrome profile.  It's hard to debug with the console because it only prints the name of the browser, and not the profile (assuming my config is not wrong).  Could it also print profile being selected (if any)?

![Screen Shot 2021-03-05 at 4 37 20 PM](https://user-images.githubusercontent.com/1675576/110176382-2fe36280-7dd1-11eb-9a2a-501dabdea0c3.png)

My config:
```
module.exports = {
  defaultBrowser: {
    name: "Google Chrome",
    profile: "Profile 2",
  },
  handlers: [
    {
      // Open apple.com and example.org urls in Safari
      match: ["mycompany.com/*", "example.org/*"],
      browser: {
        name: "Google Chrome",
        profile: "Profile 2",
      },
    },
  ],
};
```

**Comment by johnste** [comment]
You could try something like this to see when the handler is being matched!

```js
module.exports = {
  defaultBrowser: {
    name: "Google Chrome",
    profile: "Profile 2",
  },
  handlers: [
    {
      // Open apple.com and example.org urls in Safari
      match: ["mycompany.com/*", "example.org/*"],
      browser: () => {
        finicky.log("opening profile 2");
        return {
          name: "Google Chrome",
          profile: "Profile 2",
        }
      },
    },
  ],
};
```

**Comment by nynexman4464** [comment]
Interesting, ok missed that.  I ended up using `finicky.notify("Recieved url", urlString);` stolen from another example, but `.log` is better.  Still might be nice if it did that by default.

**Comment by johnste** [comment]
Yeah I agree. Will consider when I update the preferences window next.

**Comment by huyz** [comment]
@nynexman4464 How do you turn on the Finicky Console?  Can't find any docs on that

**Comment by huyz** [comment]
> @nynexman4464 How do you turn on the Finicky Console? Can't find any docs on that

Never mind, found it.  I had turned off the menubar icon long ago.  So I didn't remember it existed and I had to re-enable it to get to the console for debugging

**Comment by johnste** [comment]
Finicky 4 logs the profile used to its debug logs, hopefull this fixes the issues. https://github.com/johnste/finicky/releases/tag/v4.0.0-alpha.4
